package.json

```diff
@@ -422,7 +426,11 @@
                     }
                 }
             }
-        ]
+        ],
+        "virtualWorkspaces": {
```

Comment:
The document shows this to be under "capabilities". Is that not required?

Reply:
You're right. I had trouble getting it to show up in the extension explorer as "limited", that's probably why...  😅